unlock_indicator.h

```diff
@@ -1,7 +1,8 @@
-#ifndef _UNLOCK_INDICATOR_H
-#define _UNLOCK_INDICATOR_H
```

Comment:
Similar to the header reordering, there's no real need to change this define (I'm assuming this was a linter again)

Reply:
Macros with leading underscore are reserved for the standard libraries, I can change them back if it has to be done.

Reply:
Yeah, I personally kinda disagree with it, but it's something from upstream that I'm not too interested in changing (since if they decide to change it later, it'll make merging easier)

Reply:
(that actually goes for a lot of things - I'd been making some choices to leave some stuff as-is just to make merges / keeping up-to-date with upstream easier)